### ISSUE TITLE
Sign released binaries using PGP

### ISSUE THREAD

**Issue by jromero** [buildpacks/pack#934]
Deferred from #799

**Comment by jjbustamante** [comment]
Hi @inglor 

No yet, I talked with @natalieparellano in the past  because we did something [similar](https://github.com/buildpacks/lifecycle/pull/683) for lifecycle and I think we can do it for pack too. I think I can work on this for pack 0.35 or 0.36

**Comment by inglor** [comment]
Cosign is better but even a simple GPG sign of the tag and a list of PGP public keys who has access/permission to publish releases in the `<root>` of the repository (usually named KEYS) is enough for now. That is some effort from developers but if this is shared by a trusted small circle is OK and to my view is less effort from cosigning.

I'm looking forward to whatever approach the project chooses to strengthen the chain of supply to downstream systems / distros :)

**Comment by inglor** [comment]
@natalieparellano 

Thanks for the summary. I think you are correct and I'm mostly interested in 1st item from the list.

I had risen #2138 before I closed it in favour of this when I misunderstood the cosigning procedure. Would you like me to reopen or track everything here as one item concerning signing?

**Comment by natalieparellano** [comment]
> Would you like me to reopen or track everything here as one item concerning signing?

@jjbustamante WDYT? One issue per thing being signed might be easier to track IMO

**Comment by jjbustamante** [comment]
Yes, I agree on having issue per artifact to be signed. @inglor  we can re-open #2138 I think
